doc/backend/norms-api-v1.yaml

```diff
@@ -204,6 +204,30 @@ paths:
               schema:
                 type: string
 
+  /renders:
+    post:
+      summary: Create and return a HTML render of a provided law
```

Comment:
Just some minor feedback on the wording:
1. Typo:  "an HTML" instead of "a HTML"
2. "rendering" instead of "render" (I looked this up and "rendering" means "Darstellung / Wiedergabe"; could not find "render")
3. How about "...of the provided LDML.de XML" instead of "a provided law" - the functionality holds for all LDML, not just for laws, I suppose.

Reply:
oh true "rendering" is the noun and "render" the verb. I adjusted that (and the other wordings) :)